ios/Podfile.lock

```diff
@@ -706,4 +706,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 3ed7125d9c3d76b4d32742223aa1fd55351b31ac
 
-COCOAPODS: 1.10.1
+COCOAPODS: 1.10.0
```

Comment:
this was on odd one, I add to change it back for the Travis to pass, happy to chnge this back, but I'm not sure if I need to update something else

Reply:
Hmmm it's very odd that all the GH Actions checks were being skipped entirely (not Travis anymore btw). We shouldn't need to change this for those to run – I'm guessing it was just an incident with GH's servers, and coincidentally was resolved when you made this change.